Rocket.Chat/Views/Cells/Chat/ChatMessageCell.swift

```diff
@@ -100,9 +100,9 @@ final class ChatMessageCell: UICollectionViewCell {
     func insertGesturesIfNeeded() {
         if self.longPressGesture == nil {
             let gesture = UILongPressGestureRecognizer(target: self, action: #selector(handleLongPressMessageCell(recognizer:)))
-            gesture.minimumPressDuration = 1
+            gesture.minimumPressDuration = 0.5
```

Comment:
@rafaelks I found the default duration is 0.5 second. As for 1-second duration, it might be too slow to respond to the gesture.